extended troubleshooting in the docs and fixed a few typos (#46)
elkowar/eww: docs/content/main/configuration.md

## Configuration

For specific built in widgets `<box>, <text>, <slider>, etc` see [Widget Documentation](@/main/widgets.md)

### Placing the configuration file

> The configuration file and the scss file should lay in your `$HOME/.config/eww` folder. The XML file should be named `eww.xml` and the scss should be named `eww.scss`

Reviewer: ```suggestion
The configuration file and the scss file should lay in `$XDG_CONFIG_HOME/eww` (or, if unset, `$HOME/.config/eww`). The XML file should be named `eww.xml` and the scss should be named `eww.scss`
```
